programs/survival/easier_renewable_sponge.sc

```diff
@@ -0,0 +1,29 @@
+__on_player_attacks_entity(player,entity)->(
+	schedule(0,'__add_dead_fish',entity)//if not the fish's health isn't 0 and I cant check if its dead or not
+);
+global_puff=0;
+global_trop=0;
+global_salmon=0;
+global_cod=0;
+
+__add_dead_fish(entity)->(
```

Comment:
global_sponge_fishez = m('Pufferfish', 'Tropical Fish', ...);
..
if (has(global_sponge_fishez, str(entity)) && entity ~ 'health' == 0, ...)

Reply:
yeah, i totally understand...

Reply:
I dont know how to do the rest so ill just do it for the more obvious section